tests/test_self.py

```diff
@@ -189,6 +189,7 @@ def test_all(self) -> None:
             TextReporter(StringIO()),
             ColorizedTextReporter(StringIO()),
             JSON2Reporter(StringIO()),
+            GithubReporter(StringIO()),
```

Comment:
Ideally we would also test the output of this reporter, but I would be fine with not doing so for now as it is such a nice to have feature.

Reply:
Yes, I was thinking about tests but since Github is the one that will annotate after all, seems like we just need to trust on Github :thinking: